.github/workflows/autobuild.yaml

```diff
@@ -0,0 +1,153 @@
+            "-DZLIB_LIBRARY=/usr/lib/x86_64-linux-gnu/libz.a"
+    steps:
+    - name: Clone scripts
+      uses: actions/checkout@v2
+      with: { ref: master }
+    - name: Install tools
+      run: ${{ matrix.config.preinstall }}
+    # Visual Studio tools require a bunch of environment variables to be set.
+    # Run vcvars64.bat and re-export the current environment to the workflow.
+    # (It'd be nice to only export the variables that *changed*, oh well).
+    - name: Visual Studio environment
+      if: matrix.config.name == 'windows'
+      shell: powershell
+      run: |
+        cmd /c "`"${{ matrix.config.vcvars }}`">NUL && set" | Foreach-Object {
+          $name, $value = $_ -split '=', 2
+          if ($value) { echo "::set-env name=$($name)::$($value)" }
+        }
+    - name: Fetch target commit
+      uses: actions/download-artifact@v1
+      with: { name: release }
+    - name: Set target commit
+      run: |
+        echo ::set-env name=LLVM_COMMIT::$(cat release/commit)
+        echo ::set-env name=CLANGD_DIR::clangd_${{ github.event.release.tag_name }}
+      shell: bash
+    - name: Clone LLVM
+      uses: actions/checkout@v2
+      with:
+        repository: llvm/llvm-project
+        path: llvm-project
+        ref: ${{ env.LLVM_COMMIT }}
+    - name: CMake
+      run: >
+        mkdir ${{ env.CLANGD_DIR }}
+
+        cp llvm-project/llvm/LICENSE.TXT ${{ env.CLANGD_DIR }}
+
+        cmake -G Ninja -S llvm-project/llvm -B ${{ env.CLANGD_DIR }}
+        "-DLLVM_ENABLE_PROJECTS=clang;clang-tools-extra"
+        "-DLLVM_ENABLE_ASSERTIONS=OFF"
+        "-DLLVM_ENABLE_BACKTRACES=ON"
+        "-DLLVM_ENABLE_TERMINFO=OFF"
+        "-DCMAKE_BUILD_TYPE=Release"
+        "-DCLANG_PLUGIN_SUPPORT=OFF"
+        "-DLLVM_ENABLE_PLUGINS=OFF"
+        ${{ matrix.config.cmake }}
+    - name: Ninja
+      run: ninja -C ${{ env.CLANGD_DIR }} clangd
+    - name: Archive
+      run: >
+        7z a clangd.zip
+        ${{ env.CLANGD_DIR }}/LICENSE.TXT
+        ${{ env.CLANGD_DIR }}/bin/clangd*
+        ${{ env.CLANGD_DIR }}/lib/clang
+    - name: Upload artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: ${{ matrix.config.name }}
+        path: clangd.zip
+    - name: Upload asset
+      uses: actions/upload-release-asset@v1.0.1
+      env: { GITHUB_TOKEN: "${{ secrets.RELEASE_TOKEN }}" }
+      with:
+        upload_url: ${{ github.event.release.upload_url }}
+        asset_name: clangd-${{ matrix.config.name }}.zip
+        asset_path: clangd.zip
+        asset_content_type: application/zip
+  # Create the release, and upload the artifacts to it.
+  finalize:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+    - name: Publish release
+      run: >
+        curl -XPATCH
+        "-HAuthorization: Bearer ${{ secrets.RELEASE_TOKEN }}"
+        "https://api.github.com/repos/${{ github.repository }}/releases/${{ github.event.release.id }}"
+        "-d" '{"draft": false}'
```

Comment:
maybe set `prerelease` to `true`?

Reply:
By not setting `prerelease`, we preserve the value set when the release was originally created.